tools/import/index.html

```diff
@@ -71,6 +71,10 @@ <h2>Job Details</h2>
                     <div id="results"></div>
                 </div>
             </div>
+            <p class="button-wrapper hidden" id="stop-import-job">
+                <button type="button" class="button">Stop Job</button>
```

Comment:
I think this should be `Stop job` (lower-case j).

Reply:
Also, a default cta button may be too much for a stop action. Should it match the download button styling which adds the `accent` class?

Reply:
How about a "negative" variant?  We don't want them clicking it accidentally (there is no "Are you sure?")